problemsets-mat223/modules/module4-exercises.tex

```diff
@@ -181,5 +181,12 @@
 				real numbers.
 			\end{enumerate}
 		\end{solution}
+        
+		\prob Let $\vec u, \vec v \in \R^n$. Prove $|\vec u \cdot \vec v| \leq \|\vec u \|\|\vec v\|$, the \emph{Cauchy-Schwarz inequality}. Hint: use the geometric formula of the dot product of $\vec u, \vec v$.
+
+	        \prob Let $\vec u, \vec v \in \R^n$. Prove $\|\vec u + \vec v\| \leq \|\vec u \| + \|\vec v\|$, the \emph{triangle inequality} in $\R^n$.
+	       Hint: use the \emph{Cauchy-Schwarz inequality}.
+
+	        \prob Suppose $\mathcal {A} = \{\vec v_1,\dots , \vec v_k\} \subset \R^n$ is a set of nonzero, mutually orthogonal vectors, i.e., for all $i \neq j$, $\vec v_i \cdot \vec v_j = 0$. Prove $\mathcal{A}$ is a linearly independent set.
```

Comment:
Let's start with the definition and make this two-part. (a) just mutually orthogonal, (b) mutually orthogonal and non-zero.

Reply:
for part a), do we want the students to realize that if one of the vectors is the zero vector, $\mathcal {A}$ is not linearly independent? So I should word the question as: "Suppose $\mathcal{A}$ is a set of mutually orthogonal vectors. Is $\mathcal{A}$ a linearly independent set? Why or Why not?"

Reply:
You got it!